src/pages/philosophy/pov.mdx

```diff
@@ -29,8 +29,13 @@ Paradoxically, decisions are more distributed, and experiences can become less a
 
 It is more important than ever that we own our own ethos, make palpable our brand values, and incorporate an instantly identifiable IBMness in everything we do.
 
-> Without aesthetic, design is either the humdrum repetition of familiar clichés or a wild scramble for novelty. Without aesthetic, the computer is but a mindless speed machine, producing effects without substance, form without relevant content, or content without meaningful form.
-> – Paul Rand
+<blockquote>
```

Comment:
Can we revert these changes back? Fix in the theme should fix it and still allow for us to use the markdown formatting

Reply:
sure, done